### ISSUE TITLE
Typo in `src/window.cpp`

### ISSUE THREAD

**Issue by iperetta** [fstl-app/fstl#86]
In file `src/window.cpp`, method `Window::on_open()` (lines 181-189), there is a typo in line 184:

```
void Window::on_open()
{
    QString filename = QFileDialog::getOpenFileName(
                this, "Load .stl file", QString(), "STL files (*.stl, *.STL)"); // *** here!!
    if (!filename.isNull())
    {
        load_stl(filename);
    }
}
```

Just need to **remove last comma in line 184**, the proper string to be as: `"STL files (*.stl *.STL)"`

Otherwise, in Fedora 36 (for example), open cannot access files with extension `*.stl` (lowercase)  only uppercase ones. Actually, the current version is filtering `*.stl,` extensions which makes no sense.

**Comment by DeveloperPaul123** [comment]
@iperetta Thanks for reporting this. Would you be able to try if the suggestion in #73 works for you? Specifically [this](https://github.com/fstl-app/fstl/issues/73#issuecomment-815665547) comment.

**Comment by iperetta** [comment]
Yes, this (sur5r <https://github.com/sur5r> commented on Apr 8, 2021
<https://github.com/fstl-app/fstl/issues/73#issuecomment-815665547>) works
for Fedora 36 and Qt 5.15.5-2
---
Igor S. Peretta
***@***.***
Cel/Mob: +55 34 99277-0277
